fe/projects/Seller.Portal/src/shared/components/DynamicForm/fields/StringField.js

```diff
@@ -27,13 +27,27 @@ function StringField(props) {
     registry = getDefaultRegistry(),
     rawErrors,
   } = props;
+
+  let value = formData;
+
+  if(typeof value === "string") {
+    let re = /[!@#$%^&*(){},<>/?|'\"+=`~\[|\]|\\\\]/g;
```

Comment:
**suggestion:** let's give friendly names to variables, e.g. let regex or let regularExpression